spec.html

```diff
@@ -20537,8 +20537,9 @@ <h1>Runtime Semantics: Evaluation</h1>
             1. Let _done_ be ? IteratorComplete(_innerResult_).
             1. If _done_ is *true*, then
               1. Return ? IteratorValue(_innerResult_).
-            1. If _generatorKind_ is ~async~, then set _received_ to ? Yield(? IteratorValue(_innerResult_)).
-            1. Else, set _received_ to ? Yield(_innerResult_).
+            1. NOTE: Do not use Yield here cause Yield will wrap _innerResult_ into a IteratorValue
```

Comment:
This is not the kind of content we typically include as notes. I would just leave it out.